collect_node_diag.sh

```diff
@@ -757,6 +758,13 @@ function collect_data {
         fi
     fi
 
+    # Collect metrics from JMX for OSS C* and DDAC
+    if [ -n "$IS_COSS" ] ; then 
```

Comment:
Curious why is this specific to OSS and DDAC only. Can't we also leverage this for DSE?

Reply:
DSE (and Cassandra 4.0) has a nodetool command which allows to invoke sjk while OSS C* < 4.0 doesn't have that feature. That's why we just need this for OSS.
Using it for DSE is possible but metrics extraction is already covered.

I'm not sure about DDAC though 🤔  but using sjk shouldn't hurt there even if the nodetool sjk command is available.

Reply:
DDAC is DSE 5.1 so, I'm thinking this shouldn't be a problem at all for any version of DSE 🤔